core-bundle/src/Image/Studio/FigureBuilder.php

```diff
@@ -584,8 +584,14 @@ private function onDefineMetadata(): ?Metadata
             return null;
         }
 
+        $fileReferenceData = array_filter([
+            Metadata::VALUE_UUID => null !== $this->filesModel ? $this->filesModel->uuid : null,
+        ]);
+
         if (null !== $this->metadata) {
-            return $this->metadata;
+            return $this->metadata
+                ->with($fileReferenceData)
+            ;
```

Comment:
```suggestion
            return $this->metadata->with($fileReferenceData);
```

It's a single line in line 606 as well, so may be a single line here as well?

Reply:
The CS fixer did this. :cry: